cmd/xmtpd-cli/commands/settlement.go

```diff
@@ -0,0 +1,586 @@
+package commands
```

Comment:
nit(style): I'd probably call this file settlement_chain.go and rename appchain.go to app_chain.go, to follow go snake case standard, and for new devs opening the folder to know exactly what's going on in all these files.